minecraft/src/main/java/net/fabricmc/loader/impl/game/minecraft/McVersionLookup.java

```diff
@@ -286,8 +286,9 @@ protected static String getRelease(String version) {
 		if (matcher.matches()) {
 			int year = Integer.parseInt(matcher.group(1));
 			int week = Integer.parseInt(matcher.group(2));
-
-			if (year == 23 && week >= 51 || year >= 24) {
+			if (year == 24 && week >= 18) {
+				return "1.21";
+			} else if (year == 23 && week >= 51 || year >= 24) {
```

Comment:
This isnt correct, please follow the pattern of the other snapshots. Be careful with the year

Reply:
The year should be uncapped for the latest version, but then capped around the sepcific weeks in 2024 for 1.20.5

Reply:
Everything should now be as requested, sorry about that